doc/contents/tablesawDemo.ipynb

```diff
@@ -388,7 +395,25 @@
     "collapsed": true
    },
    "outputs": [],
-   "source": []
+   "source": [
+    "// Adding back the year column as DateColumn for charting purposes\n",
+    "DateColumn yearDates = new DateColumn(\"YearDate\");\n",
+    "for(year in summary.column('Year')){\n",
+    "    yearDates.append(java.time.LocalDate.of(year,1,1));\n",
+    "}\n",
+    "summary.addColumn(yearDates)\n",
+    "\n",
+    "plot = new TimePlot(title: 'Prie Chart for AAPL', xLabel: 'Time', yLabel: 'Max [Close]')\n",
+    "\n",
+    "yearDates = summary.column('YearDate').collect{java.sql.Date.valueOf(it)}\n",
```

Comment:
We should avoid using sql Date since we're not doing anything related to database access. We should use java.util.Date instead.